sklearn/linear_model/tests/test_huber.py

```diff
@@ -168,9 +168,7 @@ def test_huber_warm_start():
     # these would be almost same but not equal.
     assert_array_almost_equal(huber_warm.coef_, huber_warm_coef, 1)
 
-    # No n_iter_ in old SciPy (<=0.9)
-    if huber_warm.n_iter_ is not None:
-        assert_equal(0, huber_warm.n_iter_)
+    assert_equal(0, huber_warm.n_iter_)
```

Comment:
`assert huber_warm.n_iter_ == 0` ?

Reply:
Or at least I would exchange 0 and the variable to have `assert_equal(actual, desired)`

Reply:
I pushed a commit using bare asserts where appropriate.